HW12/SQLAlchemy/linguist.py

```
@@ -0,0 +1,147 @@
+    update_card.word = word
+    update_card.translation = translation
+    update_card.tip = tip
+    s.commit()
+    return Card
+
+
+def card_delete_by_id(card_id):
+    delete_card = s.query(Card).filter_by(id=card_id).first()
+    s.delete(delete_card)
+    s.commit()
+    return True if not s.query(Card).filter_by(id=card_id).all() else False
```

Ruff (0.8.2)

Use `bool(...)` instead of `True if ... else False`

Replace with `bool(...)

(SIM210)